### ISSUE TITLE
show train limit in more places

### ISSUE THREAD

**Issue by benjaminxscott** [tobymao/18xx#10276]
Currently one must change to the `Info` tab to check the current train limit, which is relevant in the context of `Game` and `Spreadsheet`, etc

Perhaps add this to the `Upcoming Trains` card (which is already present in those places)? 
That seems intuitive as the 'train related info' place

**Comment by joeyd4329** [comment]
Yes definitely related to that. And I’d agree showing the limit changes might be helpful too

**Comment by taraxvoid** [comment]
Pls unassign / close as desired
